packages/ember-intl/blueprints/ember-intl/files/__config__/ember-intl.js

```diff
@@ -44,7 +44,7 @@ module.exports = function (/* environment */) {
      * @type {String}
      * @default "'translations'"
      */
-    inputPath: 'translations',
+    inputPath: '<%= configInputPath %>',
```

Comment:
I couldn't think of a way to test this, other than publishing `ember-intl@7.0.3` and running `ember install ember-intl` in a v1 addon.

Reply:
With `ember-intl@7.0.3`, I get `inputPath: 'translations'` when I run `ember install ember-intl` in an app _and_ in a v1 addon. Not sure yet why.